### PR TITLE
Update youtube overlay design

### PR DESCRIPTION
**Pull request by abeddow91**
## What are you changing?
Updates the youtube overlay.  Updates include 
- moving the timestamp top right
- making the play button pillarless
- using the new svg play button

## Why?

This is inline with new designs that will be carried out across web (DCR, Atoms Rendering and frontend) and Apps

### CODE REVIEW

File: libs/@guardian/atoms-rendering/src/YoutubeAtomOverlay.tsx [hunk 1]
```diff
@@ -2,13 +2,12 @@ import { css } from '@emotion/react';
 import type { ArticleTheme } from '@guardian/libs';
 import {
 	focusHalo,
-	neutral,
```
Comment (abeddow91) [inline libs/@guardian/atoms-rendering/src/YoutubeAtomOverlay.tsx:5]: neutral is deprecated!

File: libs/@guardian/atoms-rendering/src/YoutubeAtomOverlay.tsx [hunk 2]
```diff
@@ -2,13 +2,12 @@ import { css } from '@emotion/react';
...
+	palette,
 	space,
 	textSans,
 } from '@guardian/source-foundations';
-import { SvgPlay } from '@guardian/source-react-components';
+import { SvgMediaControlsPlay } from '@guardian/source-react-components';
```
Comment (abeddow91) [inline libs/@guardian/atoms-rendering/src/YoutubeAtomOverlay.tsx:9]: SvgPlay has been deprecated so this PR moves to the new SvgMediaControlsPlay 👍

**Review by joecowton1 — APPROVED** [review]

**Review by joecowton1 — CHANGES_REQUESTED** [review]
@abeddow91 @sophie-macmillan we're experiencing an issue with TS 5.1 and rollup which means the latest releases of source-react-components and source-react-components-development kitchen are broken.

We've a PR in progress to roll back to TS 4.9.5 which should be merged early next week - if you can hold off merging this until then everything should be working again. Apologies for the hold up!

**Comment by joecowton1** [comment]
@sophie-macmillan @abeddow91 the fix I was talking about is now on main, if you pull that into this branch everything should work again

**Review by joecowton1 — APPROVED** [review]

File: libs/@guardian/atoms-rendering/src/YoutubeAtomOverlay.tsx [hunk 3]
```diff
@@ -73,25 +75,21 @@ const playButtonStyling = (pillar: ArticleTheme) => css`
...
-
-const videoDurationStyles = (pillar: ArticleTheme) => css`
-	${textSans.medium({ fontWeight: 'bold' })};
-	padding-left: ${space[3]}px;
-	color: ${pillarPalette[pillar][500]};
+	top: ${space[2]}px;
+	right: ${space[2]}px;
+	${textSans.xxsmall({ fontWeight: 'bold' })};
+	background-color: rgba(0, 0, 0, 0.7);
+	color: white;
```
Comment (abeddow91) [inline libs/@guardian/atoms-rendering/src/YoutubeAtomOverlay.tsx:90]:
```suggestion
	color: ${palette.neutral[100]};
```

**Pull request merged**